README.md

```diff
@@ -10,6 +10,10 @@ OpenSearch Go Client
 - [License](#license)
 - [Copyright](#copyright)
 
+> **Warning**
+> 
+> `2.3.0` is the last minor version for the 2.x line for the OpenSearch Go client. All new features will now be added to the upcoming major version 3.0.0 being developed in the [`main`](https://github.com/opensearch-project/opensearch-go) branch of this client. To upgrade to the next major version,see [UPGRADING](https://github.com/opensearch-project/opensearch-go/blob/main/UPGRADING.md). The 2.3.x line will be managed and supported for security fixes.
```

Comment:
I am not convinced me need this. At the time of writing this the latest version is 2.3.x, and when we release 3.0 the latest version will be 3.0. All major upgrades carry breaking changes. 

So what is this warning me about and what would I do differently after reading this?

If we want to say what one should use, we could add a "Stable Release" section such as https://github.com/ruby-grape/grape/#stable-release.